server/mock.js

```diff
@@ -1,11 +1,12 @@
+
 var placeholderInitiativeHeaderImage;
 var users;
 var initiatives;
 
 if (Meteor.settings.mode === 'dev') {
   console.log('======> Staging mock data...');
-  placeholderInitiativeHeaderImage = '/images/placeholder-initiative.jpg';
-  users = [];
+  placeholderInitiativeHeaderImage = '/images/placeholder-initiative.jpg',
```

Comment:
you can't do comma syntax. variables are required to be split out like they were prior to this pr